page/changelog/1.5.md

```diff
@@ -33,11 +33,11 @@ Released on June 8, 2008
 ## New bugtracker and SVN project
 
 * Dedicated bugtracker with version targeting and categories on [http://ui.jquery.com/bugs](http://ui.jquery.com/bugs)
-* Dedicated project on google code, restructured and cleaner SVN trunk and branches ([http://code.google.com/p/jquery-ui](http://code.google.com/p/jquery-ui))
+* Dedicated project on google code, restructured and cleaner SVN trunk and branches ([https://github.com/jquery/jquery-ui](https://github.com/jquery/jquery-ui))
```

Comment:
This shouldn't change. It's a historical link and the updated URL doesn't make any sense when reading the sentence.